mapstory/static/style/site/variables.less

```diff
@@ -26,4 +26,30 @@
 @gray40: mix(@black, @white, 60%);
 @gray30: mix(@black, @white, 70%);
 @gray20: mix(@black, @white, 80%);
-@gray10: mix(@black, @white, 90%);
\ No newline at end of file
+@gray10: mix(@black, @white, 90%);
+
+@gray-0: #ffffff;
+@gray-25: #fafafa;
+@gray-50: #f5f5f5;
+@gray-100: #eeeeee;
+@gray-200: #e7e7e7;
+@gray-300: #e0e0e0;
+@gray-400: #d8d8d8;
+@gray-500: #bdbdbd;
+@gray-600: #9e9e9e;
+@gray-700: #757575;
+@gray-800: #616161;
+@gray-900: #424242;
+@gray-950: #333333;
+@gray-975: #212121;
+@gray-1000: #000000;
+
+@blue-300: #0e83e8;
+@blue-500: #0f6cba;
+
+// Typography
+
+@base-font-family: 'Lato', Helvetica, Arial, sans-serif;
+@base-font-size: 1em;
+@base-font-color: @gray-950;
+@base-line-height: 1.5;
```

Comment:
I've added these vars to help keep styling consistent, but I haven't removed some of the older vars since they're still be utilized by other parts of the site. We'll have to refactor usage of the old ones out and comb through the code to get vars updated as part of a separate task. I wanted to keep the scope as narrow as possible here.